.classpath

```diff
@@ -1,52 +1,52 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
These changes should be reverted.

Reply:
I don't know how to revert this changes. Any pointers on this?